statistics/gc.go

```diff
@@ -48,22 +48,24 @@ func (h *Handle) GCStats(is infoschema.InfoSchema, ddlLease time.Duration) error
 	return nil
 }
 
-func (h *Handle) gcTableStats(is infoschema.InfoSchema, tableID int64) error {
-	sql := fmt.Sprintf("select is_index, hist_id from mysql.stats_histograms where table_id = %d", tableID)
+func (h *Handle) gcTableStats(is infoschema.InfoSchema, physicalID int64) error {
+	sql := fmt.Sprintf("select is_index, hist_id from mysql.stats_histograms where table_id = %d", physicalID)
```

Comment:
Do we need to change the `table_id` in the sql text?

Reply:
The change needs a ddl operation and may lead to incompatibility between different version of tidb server.